kubetest2/kubetest2-kind/deployer/deployer.go

```diff
@@ -59,6 +59,6 @@ func (d *deployer) DumpClusterLogs() error {
 	panic("unimplemented")
 }
 
-func (d *deployer) GetBuilder() types.Build {
+func (d *deployer) Build() error {
```

Comment:
> * minor cleanup, replace `GetBuilder` with just `Build` on the deployer interface ... deployers can just wrap a library provided default builder instead

the differences between the two patterns above is subtle, but how are going to pass the different build options from kubetest2 to the deployer - bezel, docker etc? 

the way we did it in kubetest1/kind is to pass the built type to the "constructor" of the deployer:
https://github.com/kubernetes/test-infra/blob/master/kubetest/kind/kind.go#L99

Reply:
yes, this is expected to explicitly be a property of the deployer now, different deployers wound up doing builds in `Up` in kubetest(1) because the different built in build types didn't cover their needs.

Reply:
the options will be passed in as options to the deployer, a default set of build utils will be provided to the deployers to choose to use.